test/table/tableview/table-view.spec.js

```diff
@@ -108,10 +141,25 @@ describe('Component: pfTableView', function () {
   });
 
   it('should use an htmlTemplate if one is configured', function () {
+    basicSetup();
     var nameLinks = element.find('.custom-template');
     expect(nameLinks.length).toBe(5);
     eventFire(nameLinks[0], 'click');
     expect($scope.result).toBe('You clicked on One');
   });
 
+  it('should not show pagination controls by default', function () {
+    basicSetup();
+    expect(element.find('pf-pagination').length).toBe(0);
+  });
+
+  it('should show pagination controls when configured', function () {
+    paginationSetup();
+    expect(element.find('pf-pagination').length).toBe(1);
+
+    expect(angular.element(element.find('.pagination-pf-items-current')).text().trim()).toBe('3-4');  // page # 2
+    expect(angular.element(element.find('.pagination-pf-items-total')).text().trim()).toBe('7');
+    expect(angular.element(element.find('.pagination-pf-page')).val().trim()).toBe('2');
+    expect(angular.element(element.find('.pagination-pf-pages')).text().trim()).toBe('4');
+  });
```

Comment:
Please add tests to make sure updatePageSize and updatePageNumber functions are called correctly when appropriate.